deploy/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       serviceAccountName: pravega-operator
       containers:
       - name: pravega-operator
-        image: pbelgundi/pravega-operator:m67
+        image: pravega/pravega-operator:0.5.0-rc1
         imagePullPolicy: Always
         ports:
         - containerPort: 6000
```

Comment:
Please also add pravega version "0.7.1" to version-map.yaml .... since it should be out soon...

Reply:
i have added pravega version "0.7.1" to the version-map.yaml